### ISSUE TITLE
how to set up mutt with keybase?

### ISSUE THREAD

**Issue by guaka** [keybase/keybase-issues#919]
I guess it's possible to set up mutt with keybase.
I'd like to see a basic guide for this.

(Feel free to close this issue, I'll be happy knowing that someone might add a link here at some point and that I'll be informed about that.)

**Comment by vext01** [comment]
It is true that `keybase track` imports keys into the local keychain, but (please correct me if I am wrong) tracking people is not the same as signing their key using the traditional gpg route. This means that keys imported into the keychain via keybase are both untrusted and unvalidated. This means mutt will warn you when you try to send an email to such a contact.

I've been doing some reading, and I think the whole point of keybase is that users don't need to sign each others keys. Am I right?

So I should see keybase as a method for getting people's keys, but I should still sign keys the traditional way if I want proper trust and validity?

**Comment by oconnor663** [comment]
@vext01 you're right, the purpose of Keybase is to let you trust keys without needing to bootstrap your own web of trust. If we wanted to do the most ergonomic thing possible with PGP keys, we would probably mark them with a high level of trust in the keyring right after we import them. I think that would get rid of most of the warnings you see in other programs. But PGP users are by nature conservative folks, and we worried that signing keys for you automatically would freak a lot of people out. (And then there are other questions? Are they "absolutely" trusted or what? Is this a config?) So we left it up to the user to deal with the keyring.

**Comment by JakkinStewart** [comment]
@guaka I've been working a bit with the neomutt dev and designed a script to work with keybase and mutt. Its still new right now, but its much more convenient than importing to gpg.
The link to the issue is [here](https://github.com/neomutt/neomutt/issues/53). I may write a gist for it if there's enough interest.

**Comment by zQueal** [comment]
> I may write a gist for it if there's enough interest.

I'm certainly interested! Neomutt is my prefered mail client. I assume [this is it](https://github.com/JakkinStewart/KeybaseMutt)?

**Comment by JakkinStewart** [comment]
I created a quick [gist](https://gist.github.com/JakkinStewart/eaf0451c0e78984514fad97bf0458c9a). It'll probably need some work to clarify its points, but its out and more explanatory than a mostly blank repository.
